src/test/java/com/amazon/ion/conformance/fragments.kt

```diff
@@ -154,18 +154,27 @@ private fun TestCaseSupport.readFragment(fragment: SeqElement, encoding: Encodin
 
 /** Reads an `IVM` fragment and returns a byte array with an IVM for the given [encoding]. */
 private fun TestCaseSupport.readIvmFragment(fragment: SeqElement, encoding: Encoding): Pair<ByteArray, Encoding> {
-    require(fragment.values[1].longValue == 1L)
+    val major = fragment.values[1].longValue
```

Comment:
🗺️ If the major version wasn't 1, this caused a test runner failure when constructing the test fragments rather than raising an error when running the test assertion, causing the invalid IVM tests to fail.